### ISSUE TITLE
Sponsors and join us sections of the homepage need content

### ISSUE THREAD

**Issue by zach2825** [hackgvl/hackgreenville-com#44]
I was thinking the sponsors section could explain how to become a sponsor and what it means to be a sponsor. 

the join us section might talk about some upcoming efforts and ideas we have and what people can do when they join. 

We also might be able to add a "who are we" section. Add organizers contributors and pictures of people. we could pull those images from the slack community or somewhere so they don't have to be maintained in the site.

**Comment by allella** [comment]
This would be a good one for input from the Hack Greenville admins and @pamelawoodbrowne and @jeremywight since we don't have any official sponsors.

I was thinking about adding the [all-contributors](https://allcontributors.org/docs/en/cli/configuration) to this project as some point. That integrates into the Github README and it may be possible to also have it inject into another file for showing on the website too.

**Comment by zach2825** [comment]
That's right! We talked about having a contributors file I completely forgot about that

**Comment by allella** [comment]
We have the [All Contributors block on the GitHub main page / README](https://github.com/codeforgreenville/hackgreenville-com#contributors-).

I did a quick search to see if that can be embedded into a website and nothing came up at the top of Google.

A good first issue would be to research if it's possible to embed or otherwise inject that into a page on the HG site dynamically.  If so, it may be better for the https://hackgreenville.com/about page rather than adding all those photos to every page call.

**Comment by oliviasculley** [comment]
Now that HackGreenville is getting more involved with RefactorGVL, we may want to spin the sponsor section into being able to donate into RefactorGVL's sponsorship tiers, but also providing a place for people find volunteering opportunities and venue owners to offer their venues as a meeting place, etc.

Robert has already provided a [donation link](https://hackgreenville.slack.com/archives/C3XNGDEBT/p1689041900228859) in the `#hg-labs` channel, so I think we can start moving on this after we get some answers for some questions:
1. Do we want to put a "Donate/Give" button in navbar and move donation/sponsor information to a separate page?
1. What kinds of "donations" do we want to accept? (money, a sign up link for venue owners, an email list for physical volunteering opportunities, copy about joining hackgreenville labs, etc)
1. Could we maybe have some sort of dynamic embedding of RefactorGVL sponsors into the hackgreenville footer?
1. Alternatively, just migrating all this information to the RefactorGVL page and just linking to it, etc

Hopefully this is enough to get the donation (and eventually, community resources) path conversation started!

**Comment by raiderrobert** [comment]
FWIW https://refactorgvl.com/ is a very simple site right now, and we made it very simple on purpose.

We have a lot of options, and I can help figure stuff out.

**Comment by allella** [comment]
I think it would be good to list out what we could do on this page and then figure out what we should do for the near term.

**Volunteer Ideas** 

1) I was imagine we could put a list of links to volunteer opportunities for related organizations. Synergy Mill has clean up days and such, so that's one idea.  Putting a link to those organization's calendar would be less work than trying to maintain a current list of events.

Unless events are put into an API, and tagged, then it would be hard to pull them from Meetup or Eventbrite. Meetup has tags, but I looked and didn't see where the tags reflected in the legacy API we're using. They may be an option if we switched to the newer Meetup API, but I didn't look and that requires a Pro account.  Also, even with tagging, we'd still be beholden to the event creator remembering to tag their event with "Volunteer" or some such.  The alternative, if we wanted to automate it, would be to look for certain words in the events of certain groups and assume those words = volunteer opportunities, but that's not keeping things simple, so not really jazzed about that idea.

2) There are other as needed volunteer things, like Christina's Hack For Good summer school program, but we'd have to add those things manually and remember to remove them.

**Donation Ideas**

1) Link to RefactorGVL, DevelopGVL, and Synergy Mill as donation options. There may be other no profits in our orgs list, like the Imagine Upstate, so we'll have to look through the list. I would assume we'd limit our potential orgs to ones who we're covering in the orgs API, but there could well be exceptions.

2) Once there's a formal HG + RefactorGVL agreement, that feels like the better time to consider highlighting RefactorGVL. Though, I'm not sold on splashing a donate QR code on the HackGreenville site for just one entity. It may be more respectful to the other non-profits we're highlighting to give RefactGVL a bit more styling on the donation page with a note about the relationship of HG being a program.

**Sponsors**
1) There are some sponsors on the data.openupstate.org, but those are CFG artifacts from OpenWorks hosting the group and the HackGVL logo was previously a CFG logo.  I'm not set on needing that site to keep any sponsors for now, so we could even remove OpenWorks, and leave the HackGreenville Labs under the "High Fives and Six Packs". 

Eventually I could see that data site being under an HG subdomain, so we might want to have any HG sponsors also listed on the bottom of that page.

2) I don't think we need to put sponsors on the HG site for now. Zach created this issue in 2020 when we did a rebuild and I think he just assumed we had HG sponsors, but we're never really had any besides people sponsoring a lunch or holiday party. It seems if we're dropping logos on websites, then it would be as part of some sponsorship packages RefactorGVL would want to define.
